.github/workflows/check-manifests.yml

```diff
@@ -18,6 +18,10 @@ jobs:
     - name: Check out code
       uses: actions/checkout@v4
 
+    - name: Define missing rpm macros
+      run: |
+        [[ -n $(rpm --eval '%bcond test 1') ]] && echo '%bcond() %[ (%{2}) ? "%{expand:%%bcond_without %{1}}" : "%{expand:%%bcond_with %{1}}" ]' > ~/.rpmmacros
```

Comment:
Thanks for adding this workaround, @ddstreetmicrosoft! Could you only add a short comment explaining that this is because we run these checks on a Ubuntu image with RPM version < 4.18.0, where `%bcond` was introduced?

Reply:
ack